File: BAEKJOON/java/16017_Telemarketer_or_not.java
import java.io.*;

public class Main {
    public static int[] number = new int[4];

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        input();
        solution();

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
       for (int i = 0; i < 4; i++) {
           int n = Integer.parseInt(br.readLine());
           number[i] = n;
       }
    }

    private static void solution() throws IOException {
        if (number[0] != 8 && number[0] != 9) {
            bw.write("answer");
            return;
        }

        if (number[3] != 8 && number[3] != 9) {
            bw.write("answer");
            return;
        }

        if (number[1] != number[2]) {
            bw.write("answer");
            return;
        }

        bw.write("ignore");
    }
}

File: BAEKJOON/cpp/16017_Telemarketer_or_not.cpp
#include <iostream>
using namespace std;

int number[4];

void input() {
	for (int i = 0; i < 4; i++) {
		cin >> number[i];
	}
}

void solution() {
	if (number[0] != 8 && number[0] != 9) {
		cout << "answer";
		return;
	}

	if (number[3] != 8 && number[3] != 9) {
		cout << "answer";
		return;
	}

	if (number[1] != number[2]) {
		cout << "answer";
		return;
	}

	cout << "ignore";
}

int main() {
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	input();
	solution();

	return 0;
}